apps/src/weblab/WebLab.js

```diff
@@ -217,6 +244,27 @@ WebLab.prototype.onIsRunningChange = function () {
 };
 
 /**
+ * Load the asset list and store it as this.currentAssets
+ */
+WebLab.prototype.loadCurrentAssets = function () {
+  assetsApi.ajax('GET', '', (xhr) => {
+    assetListStore.reset(JSON.parse(xhr.responseText));
+    this.currentAssets = assetListStore.list().map(function (asset) {
```

Comment:
This is a good arrow function candidate (`map` usually is), in this case with an object expression `({})` instead of a block `{}` on the right-hand side (this is [airbnb preferred](https://github.com/airbnb/javascript#arrows--implicit-return)):

```javascript
this.currentAssets = assetListStore.list().map(asset => ({
  name: asset.filename,
  url: '/v3/assets/' + dashboard.project.getCurrentId() + '/' + asset.filename
}));
```